receptor_satellite/worker.py

```diff
@@ -1,8 +1,50 @@
-import asyncio
 import aiohttp
-# import functools
+import asyncio
 import json
 
+# TODO: Get this from somewhere
+SATELLITE_HOST = 'localhost:3000'
+
+
+class SatelliteAPI:
```

Comment:
None of the functions defined under this class are actually methods (they don't take an initial self parameter).  Do you really need this class?

Reply:
used the class just as a logical grouping

Reply:
In that case, you should decorate the functions with `@staticmethod`

Reply:
Moved to a separate file, no decorator should be needed there I guess